### DISCUSSION TITLE
Is it possible to exclude localhost from tracking

### DISCUSSION THREAD

**Discussion by jarvis09-yann** [umami-software/umami#2713] [Q&A] [answered]
Hi, I'd like to know if it was possible to not track localhost (127.0.0.1) because I don't want to pollute my metrics by showing all the traffic generated by me working locally on my website.

As you can see in the screenshot below, a good portion of the traffic comes from my development server.
![image](https://github.com/umami-software/umami/assets/72513680/10a900af-2106-4511-89c9-b4b32780fac6)

Thanks!

**Comment by jarvis09-yann** [comment]
im still looking for help

**Comment by L33Z22L11** [comment, marked as answer]
see https://github.com/umami-software/umami/discussions/852#discussioncomment-9485629

**Reply by jarvis09-yann** [reply]
Thanks!
